en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -472,4 +473,75 @@ To create a For Each Loop:
 
 You can add logic and actions in different patterns, depending on what your flow needs to do.
 
+== Designate Error Handling 
+
+When executing a flow, you can watch for errors in connection or logic in specific steps of your flow and then designate logic to handle the errors that may occur. You can also designate steps to complete additional actions or to send notifications in the event of an error. 
+
+[[access-error-handling]]
+=== Access Error Handling 
+
+To access the Error Handler:
+
+. <<create-a-new-flow,Create a new flow>> or open an existing flow.
+. If you create a new flow:
```

Comment:
You could make this a bullet sublist of step 1 (since you only complete one of the sublists).
1. blah blah:
  - New flow:
     a. yadda
  - Existing flow:
     b. blah

Reply:
Done!